imports/plugins/core/cart/server/no-meteor/mutations/convertAnonymousCartToNewAccountCart.js

```diff
@@ -39,7 +41,8 @@ export default async function convertAnonymousCartToNewAccountCart({
   };
 
   if (anonymousCart.referenceId) {
```

Comment:
Since you have it destructured now, this `if` could be changed to `if (referenceId)`